### PR TITLE
chore: version details and history

### PR DESCRIPTION
**Pull request by mbtools**
The update to the readme and a table with the historical details clarifies the current state. I understand that there will be some consolidation of versions/branches. I will update the table when the time comes.

I think you should update the npm tags a bit. 

Currently:

```bash
> npm dist-tag ls verdaccio
4-next: 4.13.2
latest-6: 6.0.0-rc.1
latest: 6.0.1
next-7: 7.0.0-next-7.20
next-8: 8.0.0-next-8.3
previous: 4.12.2
```

Suggested:

```bash
> npm dist-tag rm verdaccio 4-next
> npm dist-tag rm verdaccio previous
> npm dist-tag rm verdaccio latest-6
> npm dist-tag add verdaccio@6.0.1 latest-6
> npm dist-tag add verdaccio@5.32.2 latest-5
# and if you like for consistency...
> npm dist-tag add verdaccio@4.13.2 latest-4
> npm dist-tag add verdaccio@3.13.1 latest-3
> npm dist-tag add verdaccio@2.7.4 latest-2
```

### CODE REVIEW

**Review by juanpicado — APPROVED** [review]
Thank u :)

File: VERSIONS.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+The following table describes the versions of this project:
+
+| Version          | Supported                           | Minimum Node.js | Branch | Npm Tag            | Docker Images             |
+| ---------------- | ----------------------------------- | --------------- | ------ | ------------------ | ------------------------- |
+| 4.x              | :x: (deprecated)                    | 10              | 4.x    | latest-4           | 4, 4.x, 4.x.x, 4.x-next   |
+| 5.x previous     | :white_check_mark: (until Nov 2024) | 14              | 5.x    | latest-5           | 5, 5.x, 5.x.x, 5.x-next   |
+| _6.x current_    | :white_check_mark:                  | _18_            | _6.x_  | _latest-6, latest_ | _6, 6.x, 6.x.x, 6.x-next_ |
+| 7.x next         | :x:                                 | 18              | 7.x    | next-7             | 7.x-next                  |
+| 8.x experimental | :x:                                 | 18              | master | next-8             | nightly-master            |
```
Comment (juanpicado) [inline VERSIONS.md:11]: I just noticed, the README also need an small update to use `next-8` instead `next`

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 > Looking for Verdaccio version 5 or 6? Version 6 is the latest version and successor to version 5. Version 6 requires Node.js 18 or higher and is maintained in the `6.x` branch.
 
-> The plugins for versions 5 and 6 are located at the [`verdaccio/monorepo`](https://github.com/verdaccio/monorepo) repository. Plugins for the `next` version are hosted in this project under the `./packages/plugins` folder.
+> The plugins for versions 5 and 6 are located at the [`verdaccio/monorepo`](https://github.com/verdaccio/monorepo) repository. Plugins for the `next-8` version are hosted in this project under the `./packages/plugins` folder.
```
Comment (juanpicado) [inline README.md:13]: A bit below there is `npm install -g verdaccio@next` 🙈

Comment (mbtools) [reply README.md:13]: 👍

**Comment by juanpicado** [comment]
Thanks, I'll keep it here until I run the `dist-tags` but all LGTM

**Comment by mbtools** [comment]
I marked 5.x as deprecated. I'm not sure what you mean by "docker version". Do you mean the node version used to build the docker images?

**Comment by juanpicado** [comment]
> I marked 5.x as deprecated. I'm not sure what you mean by "docker version". Do you mean the node version used to build the docker images?

Docker does not follow the same versioning conventions, for instance:

https://hub.docker.com/r/verdaccio/verdaccio/tags

User then can chose between use `5.32.0` or `5.32` or `5`, but also if they want to have quick updates can use `5.x-next` or `6.x-next`, `7.x-next` or `nightly-master` for the experimental.

Furthermore, we have Heml charts, which does not follow neither Docker or npmjs conventions, rather pure semantic versioning without any connection with 5.x or 6.x

https://artifacthub.io/packages/helm/verdaccio/verdaccio

I cannot deprecate docker tags or helm tags, so I think would be useful to have both on this document just for transparency. What do you think?

**Review by juanpicado — APPROVED** [review]

**Pull request merged**